grunt/clean.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
```

Comment:
maybe it's my version of grunt, but the `<%= %>` templates were not putting the grunt config in the template context.  pulling hair out!